packages/form-js-editor/src/rendering/PropertiesPanel.js

```diff
@@ -293,7 +324,7 @@ function ColumnsProperty(props) {
 
   return (
     <div class="fjs-properties-panel-entry">
-      <Number label="Columns" onInput={ onInput } value={ value } min="1" max="3" />
+      <Number id="columns" label="Columns" onInput={ onInput } value={ value } min="1" max="3" />

```

Comment:
Is this a real DOM `id`? If so, we could generate unique IDs using a per-editor instance unique identifier.

Reply:
Yes, this will end up being the ID used for the label's `for` attribute and the input's `id` attribute.

Reply:
Let's keep the instance based IDs in the back of our head.

Reply:
A pattern I've seen which circumvents this issue completely is to wrap the input in a `<label>` tag all together: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

```
<label>Do you like peas?
  <input type="checkbox" name="peas">
</label>
```

This is supposed to work for all kinds of inputs, not only checkboxes.